streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -120,16 +112,14 @@ public Producer<byte[], byte[]> getProducer(final Map<String, Object> config) {
         "threadId",
         clientSupplier,
         null,
-        null,
         logContext,
         mockTime
     );
-    final StreamsProducer eosAlphaStreamsProducerWithMock = new StreamsProducer(
-        eosAlphaConfig,
-        "threadId",
+    final StreamsProducer eosBetaStreamsProducerWithMock = new StreamsProducer(
+        eosBetaConfig,
```

Comment:
```suggestion
    final StreamsProducer eosV2StreamsProducerWithMock = new StreamsProducer(
        eosV2Config,
```
We want to get rid of the "beta".

Reply:
I wanted to keep such renamings out of the first PR to keep it reviewable (and added a note above), but maybe it was ill-conceived in this case. Done.